RELEASE/scripts/autoscend.ash

```diff
@@ -4836,6 +4842,8 @@ boolean doTasks()
 	# This function buys missing skills in general, not just for Picky.
 	# It should be moved.
 	picky_buyskills();
+	boris_buySkills();
```

Comment:
> The skill buying functions for those two paths were in redundant nested functions whose only functionality was to call on buyskills then return false. So cleaned that up.

Eh, leave it the way it is. That's the perfect place to put later Boris-specific code that you need without cluttering up `autoscend.ash`.

Reply:
While true, at the moment there is only the 1 thing in it. And that 1 thing is clearer if it is placed with the other path_buySkills functions.
We can always reinstate it if we actually come up with something to put in it. But are we? boris is a pretty old path and none of the improvements I have thought up for it require that function.

Reply:
That `LM_pathname()` convention is used in a lot of places. I'd prefer to leave the code open to extension and avoid making non-functional changes that buck the codebase's prior convention.

Reply:
requested change made